### ISSUE TITLE
TypeError in pydotprint

### ISSUE THREAD

**Issue by Spaak** [pymc-devs/aesara#309]
As part of the larger effort of moving over PyMC3's graph printing (and string representations) to v4 (see https://github.com/pymc-devs/pymc3/issues/4494), I encountered the following issue:

```
Traceback (most recent call last):
  File "/home/eelke/python/tmp.py", line 35, in <module>
    pydotprint(x, '~/tmp.pdf')
  File "/home/eelke/repos/Theano-PyMC/aesara/printing.py", line 983, in pydotprint
    nw_node = Node(aid, label=astr, shape=apply_shape)
TypeError: Node() takes no arguments
```

when executing the following:

```
import pymc3 as pm
from aesara.printing import pydotprint, debugprint

model = pm.Model()
with model:
    x = pm.Normal('x', mu=0, sigma=1)

pydotprint(x, '~/tmp.pdf')
```

In case it's relevant:

```
>>> debugprint(x)
normal_rv.1 [id A] 'x'   
 |RandomStateSharedVariable(<RandomState(MT19937) at 0x7FC3AF2FEA40>) [id B]
 |TensorConstant{[]} [id C]
 |TensorConstant{11} [id D]
 |TensorConstant{0} [id E]
 |TensorConstant{1.0} [id F]
```

Aesara is at current `master` (5795760d3d9f160fca3d20b266a224e87d86a6b4), PyMC3 is at current `v4` (main repository, pymc-devs/pymc3@932672a6995647052224f9ef0782b8bfb78bd8e7).

**Comment by Spaak** [comment]
This is not related to anything PyMC3-specific, as I get the same error with the following:

```
import aesara as ae
import aesara.tensor as aet
from aesara.printing import pydotprint, debugprint

x = aet.as_tensor_variable(1.0)
y = aet.as_tensor_variable(2.0)
z = y*x
pydotprint(z, '/home/eelke/tmpgraph')
```

**Comment by michaelosthege** [comment]
This bug was released in Theano-PyMC 1.1.2, but we're not planning to make a patch release of Theano-PyMC.

Instead, I'm going to create a PR for PyMC3 `v3.11.3` that applies the following hotfix automatically:

```python
import pydot
import theano.printing

theano.printing.Node = pydot.Node
```
